packages/react/src/combobox/ComboboxBase/ComboboxBase.tsx

```diff
@@ -131,50 +129,43 @@ export function ComboboxBase<Option extends DefaultComboboxOption>({
 							onClick={downshift.reset}
 						/>
 					)}
-					<div
```

Comment:
I've combined this `div` and `ComboboxList` (ul) component into the same component which is more inline with the ARIA patterns 🎉